lib/client.js

```diff
@@ -1,29 +1,22 @@
 (function (global, factory, undefined) {
 
-  if ( typeof module === 'object' && typeof module.exports === 'object' ) {
-    var ActivityStreams = require('activity-streams');
+  if ( typeof module === "object" && typeof module.exports === "object" ) {
```

Comment:
@galfert looks interesting will give it a read. Though I'm not sure how we can qualify the term "most people" :)